### PR TITLE
Flipping cards

### PR DESCRIPTION
**Pull request by Monika-Marciniak**

### CODE REVIEW

File: memorygame.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <title>Document</title>
+  <link rel="stylesheet" href="memorygame.css">
+</head>
+<body>
+  <div class="container">
+  <div class="card" id="cardId">
+  <div class="cover front">
+  <img src="./pictures/puppy.jpg" alt=""></div>
+  <div class="cover back">
+  <img src="./pictures/doggo.jpg" alt=""></div>
```
Comment (shpomp) [inline memorygame.html]: The task is to "display a single card on the page using DOM manipulation." - which is not what is done here. 
Refactor and implement the actual task as described 🙏🏽

**Review by shpomp — CHANGES_REQUESTED** [review]
This looks unfinished, I am not sure if that was meant to be. 🤔 
The first task is not implemented. 
Additionally, the card does not look like a card - it is just 2 images being exchanged, they have different sizes and neither of them looks like a neutral backside. 
The toggle function application is great, though, so you are close! 👍🏽

File: memorygame.html (hunk 1 above)
Comment (Monika-Marciniak) [reply memorygame.html]: I changed the code, please see if its okay. I have some issues with my vsc so i sent u a link to playground too

**Comment by shpomp** [comment]
@Monika-Marciniak  It is not the best idea to merge code without getting it checked and reviewed when your first version was not working. 
This merged code does not work too - there is nothing on the screen (for a very simple reason 👀 ). 
So you are merging non-functional code as the main version of your app and perhaps you don't want to do that. Imagine if this was a real app that was online! 🧨 😀 
Please run and test thoroughly before you merge, and, preferably, ask for feedback once you are ready with the tasks of the week - this way you can have a consistent process and a working app!

**Pull request merged**